### PR TITLE
Temporary manual update for tv tutorial ipynb

### PR DESCRIPTION
**Pull request by vfdev-5**
Description:
- https://github.com/pytorch/tutorials/pull/2540#issuecomment-1705037881

TL;DR: We need torchvision nightly to run correctly python tutorial from https://github.com/pytorch/tutorials/pull/2540 and according to @NicolasHug, we can't install torchvision nightly to the CI. He suggested to update the colab notebook (this PR)

Here we send a PR to gh-pages branch directly same as previously torchvision tutorial notebook was pushed

After v0.16 release we can remove this notebook once https://github.com/pytorch/tutorials/pull/2540 is merged

Result:

- https://colab.research.google.com/github/pytorch/tutorials/blob/update-tv-tut-ipynb-gh-pages/_downloads/torchvision_finetuning_instance_segmentation.ipynb

### CODE REVIEW

**Review by NicolasHug — COMMENTED** [review]
Thanks for the PR @vfdev-5 . Can you please update this notebook with the same changes made in https://github.com/pytorch/tutorials/pull/2546 so we can merge it? Thanks!

**Comment by vfdev-5** [comment]
@NicolasHug just synced it, you can see the result here: https://colab.research.google.com/github/pytorch/tutorials/blob/update-tv-tut-ipynb-gh-pages/_downloads/torchvision_finetuning_instance_segmentation.ipynb
